src/main/com/example2/radek/CharacterCounter.java

```diff
@@ -0,0 +1,24 @@
+package main.com.example2.radek;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class CharacterCounter {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Insert word, which you want to count: ");
+        String Word = scanner.nextLine();
```

Comment:
promenna Word, Input ... ty promenne malym na zacatku ... v jave velkym zacinaji objekty, jak to ctu, tak se mi to porad plete

Reply:
Ok, objekty v javě jsou brány ty, které pracují s UI ? Jen abych si to nemyslel špatně, u PB existuje visual/non-visual object tzn. vše co je na frontendu, ale něco je pro oko uživatele skryto z nějakého důvodu.

Reply:
Je to úplně jinak. K tomu se brzo dostaneme.
Pro začátek se představ, že každý Java file je objekt.
Jestli máš nějaký decompiler v idei, tak podrž ctrl najed na String a rozklikni. Uvidíš vnitřní implementaci objektu String

Reply:
Trochu rozumím, ale určitě si o tom zjistím více